serv/backend/dtos/user.dtos.go

```diff
@@ -0,0 +1,36 @@
+// SPDX-FileCopyrightText: 2024 The JoynRide Authors and CCOS-USP <https://ccos.icmc.usp.br>
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+package dtos
+
+type User struct {
+	Id                    string `json:"id"`
+	Name                  string `json:"name"`
+	Phone                 string `json:"phone"`
+	Address               string `json:"address"`
+	Gender                string `json:"gender"`
+	Birth_date            string `json:"birth_date"`
```

Comment:
Looks great! But maybe we should stick to CamelCase to keep things consistent, i.e., instead of `Birth_date`, we use `BirthDate`.

Actually, while I was searching for the usual naming convention for struct fields in Go (which seem to be indeed CamelCase), I discovered that the community has some [strong opinions on the subject](https://github.com/golang/go/issues/64461) lol

![image](https://github.com/user-attachments/assets/511c89e4-2f62-46aa-bc1b-7a2f3eec9e2e)
🔥